### ISSUE TITLE
Poor Color Contrast is Difficult to See

### ISSUE THREAD

**Issue by flywire** [ma4nn/pp-terminal#2]
![Image](https://github.com/user-attachments/assets/0bf2e23c-507d-419c-be99-a3315364f65c)

---

https://www.accessibility.com/glossary/advanced-perceptual-contrast-algorithm
> The WCAG 2.x contrast methods are being replaced for WCAG 3.0 APCA.

https://www.achecks.org/apca-accessible-colour-contrast-checker/

**Comment by ma4nn** [comment]
These are the default colors for help texts of the used [Typer](https://typer.tiangolo.com/) library. 

To disable the colors you can create the environment variable `NO_COLOR` with any value (see also [here](https://no-color.org/)). Then the console output will only be black and gray.
I will update the README file to include that info.

**Comment by flywire** [comment]
That's great.
